src/invidious/yt_backend/youtube_api.cr

```diff
@@ -46,39 +50,51 @@ module YoutubeAPI
   # List of hard-coded values used by the different clients
   HARDCODED_CLIENTS = {
     ClientType::Web => {
-      name:       "WEB",
-      name_proto: "1",
-      version:    "2.20240304.00.00",
-      screen:     "WATCH_FULL_SCREEN",
-      os_name:    "Windows",
-      os_version: WINDOWS_VERSION,
-      platform:   "DESKTOP",
+      name:            "WEB",
+      name_proto:      "1",
+      version:         "2.20240304.00.00",
+      screen:          "WATCH_FULL_SCREEN",
+      os_name:         "Windows",
+      os_version:      WINDOWS_VERSION,
+      browser_name:    "Chrome",
+      browser_version: "128.0.0.0",
```

Comment:
can you put that in a constant at the top? It will be easier to maintain in the future!